packages/legacy-workbench/src/js/angular/controllers.js

```diff
@@ -284,6 +286,22 @@ function mainCtrl($scope, $menuItems, $jwtAuth, $http, toastr, $location, $repos
         }
     });
 
+    const subscribeToCookieConsentChanged = () => {
+        document.body.addEventListener(COOKIE_CONSENT_CHANGED_EVENT, updateCookieConsentHandler)
+        return () => document.body.removeEventListener(COOKIE_CONSENT_CHANGED_EVENT, updateCookieConsentHandler);
+    }
+
+    function updateCookieConsentHandler(consentChangedEvent) {
```

Comment:
This function would be hoisted as it is now, but better move it before the subscription above.